independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/util/SseMediaType.java

```diff
@@ -0,0 +1,18 @@
+package org.jboss.resteasy.reactive.common.util;
+
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Representation of the media types suitable for server streaming.
+ */
+public final class SseMediaType {
```

Comment:
I disagree with that name, JSON streaming isn't SSE (Server-Sent Events) which is its own spec. We could however call it `StreamMediaType`.

Or better yet we make our own subtype of `MediaType` that is `RestMediaType` which has these additional types?

Reply:
Renamed to `StreamMediaType`. 
I didn't rename to `RestMediaType` because using `RestMediaType.APPLICATION_NDJSON` sounds weird to me. Maybe `ReactiveMediaType`?

Reply:
And making it a subtype of `MediaType` allows us to just tell everyone to use `RestMediaType` for all their media types.

Reply:
Okis, will rename it to `RestMediaType`.

Reply:
Renamed to `RestMediaType`.